homeassistant/components/modbus/new_config.txt

```diff
@@ -0,0 +1,45 @@
+This PR contains the new configuration for all devices.
+
+In order to keep track of the changes, this document contains a chapter
+for each device type, indicating the new configuration and how to convert
+from old configuration.
+
+binary_sensor:
+
+	modbus:
+		binary_sensors: []
+
+	--> NO CHANGES.
+
+
+sensor:
+	modbus:
+		sensors: []
+
+	new parameters:
+		CONF_INPUT_TYPE, optional (CALL_TYPE_COIL) CALL_TYPE_DISCRETE or CALL_TYPE_COIL
```

Comment:
The old parameter was "register type" with values: holding or input. I guess this is just a typo. Sensor doesn't use coils or discrete inputs.

Reply:
Good catch (Copy/paste is a dangerous tool). Removed the file, since it was merely there so you could see the changes easily.